istio.VERSION

```diff
@@ -15,3 +15,11 @@ export FORTIO_HUB="docker.io/istio"
 export FORTIO_TAG="0.6.9"
 export HYPERKUBE_HUB="quay.io/coreos/hyperkube"
 export HYPERKUBE_TAG="v1.7.6_coreos.0"
+export STATSD_HUB="prom/statsd-exporter"
+export STATSD_TAG="v0.5.0"
```

Comment:
It looks like some of the original cases were using "latest" while others were using "v0.5.0".  I have no idea on which is more appropriate but I'll just point it out in case another reviewer has more knowledgeable thoughts on this.

Reply:
Yeah, this looked confusing to me. If someone can clarify, I will add the right tag.